settings.py

```diff
@@ -347,7 +347,7 @@
 KIBANA_ELASTICSEARCH_USER = os.environ.get('KIBANA_ELASTICSEARCH_USER', 'kibana')
 KIBANA_ELASTICSEARCH_PASSWORD = os.environ.get('KIBANA_ES_PASSWORD')
 
-HAIL_BACKEND_SERVICE_HOSTNAME = os.environ.get('HAIL_BACKEND_SERVICE_HOSTNAME')
+HAIL_BACKEND_SERVICE_HOSTNAME = os.environ.get('HAIL_BACKEND_SERVICE_HOSTNAME', 'hail-search')
```

Comment:
Yet another question here, is it better practice to default to `localhost` for both this and the pipeline runner server?

Reply:
so my understanding is that using docker-compose things are named the service name by default not localhost

Reply:
👍